src/actinia_core/resources/common/user.py

```diff
@@ -361,11 +361,6 @@ def __str__(self):
 
         return(string)
 
-    def list_all_users(self):
-        """List all users in the database
-        """
-        return self.db.list_all_ids()
-
```

Comment:
Are these lines deleted intentionally?

Reply:
Yes, it was defined twice:

https://github.com/mundialis/actinia_core/blob/e451a7301409d5c37463b8c2c875d9cb04c69b29/src/actinia_core/resources/common/user.py#L364

https://github.com/mundialis/actinia_core/blob/e451a7301409d5c37463b8c2c875d9cb04c69b29/src/actinia_core/resources/common/user.py#L593

Reply:
Thanks for the clarification.